epochStart/metachain/systemSCs.go

```diff
@@ -251,7 +257,7 @@ func (s *systemSCProcessor) ProcessSystemSmartContract(
 	}
 
 	if s.flagCorrectNumNodesToStake.IsSet() {
-		err := s.cleanAdditionalQueue()
+		err := s.cleanAdditionalQueue() // TODO: Deactivate this?
```

Comment:
we need this for backwards compatibility reasons. the TODO can be safely removed :D

Reply:
Yes, left this as a `TODO` for the next PR, but there it is already taken care of. Will remove it